### PR TITLE
Adds jenkins build/publish/pipeline scripts to utils repo

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**
* The `pipeline.jenkins` scripts drive the Jenkins pipeline, which calls the Build and Promote job
* The `build.jenkins` script builds a utils compliant project and generates a pax.Z artifact
* The `publish.jenkins` script publishes a pax.Z to as a github release

https://128.168.139.253:8443/view/Framework/ has been updated to call these scripts

### CODE REVIEW

File: cicd/build.jenkins [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Assumes utils repo is cloned into PWD
```
Comment (MikeFultonDev) [inline cicd/build.jenkins:1]: This is 'groovy' code correct?

File: cicd/build.jenkins [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export NO_COLOR=1
+
+# source bootstrap environment variables on zot
+. /jenkins/.env
+
+# Add cloned utils dir to PATH
+export PATH="$PWD/bin:$PATH"
+
+# Get port name based on git repo
+PORT_NAME=$(basename "${PORT_GITHUB_URL}")
```
Comment (MikeFultonDev) [inline cicd/build.jenkins:16]: shouldn't this be PORT_GIT_URL and not PORT_GITHUB_URL ?

File: cicd/build.jenkins [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Assumes utils repo is cloned into PWD
+
```
Comment (MikeFultonDev) [inline cicd/build.jenkins:2]: can you add a description? I'm not sure which env vars are set in /jenkins/.env and which (if any) are set before-hand? This appears to also only work for a GIT port and not a TARBALL port.

File: cicd/build.jenkins [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# Get port name based on git repo
+PORT_NAME=$(basename "${PORT_GITHUB_URL}")
+PORT_NAME=${PORT_NAME%%.*}
+PORT_NAME=${PORT_NAME%%port}
+
+# Set install dir to workspace/install
+export PORT_INSTALL_DIR="${PWD}/install"
+mkdir -p ${PORT_INSTALL_DIR}
+
+git clone -b "${BRANCH}" "${PORT_GITHUB_URL}" ${PORT_NAME} && cd ${PORT_NAME}
```
Comment (MikeFultonDev) [inline cicd/build.jenkins:24]: how is BRANCH related to the PORT_GIT_BRANCH ?

File: cicd/pipeline.jenkins [hunk 5]
```diff
@@ -0,0 +1,18 @@
+def repo = params.get("REPO")
```
Comment (MikeFultonDev) [inline cicd/pipeline.jenkins:7]: As with the other jenkins files, a description would be good. Do we want to have some sort of correlation between 'sh' env vars and these jenkins vars? Should they be consistently named?

**Comment by IgorTodorovskiIBM** [comment]
Closing in favour of https://github.com/ZOSOpenTools/meta/pull/11

**Pull request closed without merging**